fixtures/compatibility/README.md

```diff
@@ -2,6 +2,8 @@ This directory contains sample repositories from different versions of Notary cl
 
 Notary client makes no guarantees of future-compatibility though (that is, repositories produced by newer clients may not be able to be read by old clients.)
 
+Backwards compatibility has been tested in ../../client/backwards_compatibility_test.go
```

Comment:
nit: use backticks (like `) to pretty print this.  Also I think just `client/backwards_compatibility_test.go` is enough detail